app/views/layouts/application.html.erb

```diff
@@ -6,17 +6,26 @@
     <title><%= t('appname') %></title>
     <%= csrf_meta_tags %>
 
-    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <% if (action_name === "new" && controller_name == "sessions" or controller_name === "users" && action_name == "new" or controller_name === "users" && action_name == "create") %>
+        <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track'=>'reload' %>
+        <%= stylesheet_link_tag 'navbar', media: 'all', 'data-turbolinks-track'=>'reload' %>
```

Comment:
navbar.css tag could be put in navbar.html.erb and this condition left out.

Reply:
But then the style link would be inside the body, which is at least bad style.
But we can move the application tag outside of the conditional